### PR TITLE
Use Cognitive Services

### PR DESCRIPTION
**Pull request by nicolastarzia**
I created a class to use cognitive services for text comment.

### CODE REVIEW

File: JekyllBlogCommentsAzure/PostCommentToPullRequestFunction.cs [hunk 1]
```diff
@@ -111,7 +111,15 @@ private static bool TryCreateCommentFromForm(NameValueCollection form, out Comme
                 errors.Add("email not in correct format");
 
             comment = errors.Any() ? null : (Comment)constructor.Invoke(values.Values.ToArray());
-            return !errors.Any();
+            var notHasErrors = !errors.Any();
```
Comment (damieng) [inline JekyllBlogCommentsAzure/PostCommentToPullRequestFunction.cs]: Can we call this variable isFormValid?

File: JekyllBlogCommentsAzure/PostCommentToPullRequestFunction.cs [hunk 2]
```diff
@@ -111,7 +111,15 @@ private static bool TryCreateCommentFromForm(NameValueCollection form, out Comme
...
+
+            if (notHasErrors)
```
Comment (damieng) [inline JekyllBlogCommentsAzure/PostCommentToPullRequestFunction.cs]: I think for backward compatibility we should also include `&& !String.IsNullOrWhitespace(ConfigurationManager.AppSettings["SentimentAnalysis.SubscriptionKey"])` so existing users who haven't yet setup the sentiment analysis key can continue working as-is.

File: JekyllBlogCommentsAzure/PostCommentToPullRequestFunction.cs (hunk 1 above)
Comment (nicolastarzia) [reply JekyllBlogCommentsAzure/PostCommentToPullRequestFunction.cs]: Should be awesome!

File: JekyllBlogCommentsAzure/PostCommentToPullRequestFunction.cs (hunk 2 above)
Comment (nicolastarzia) [reply JekyllBlogCommentsAzure/PostCommentToPullRequestFunction.cs]: Ok, I added this validation for backward compatibility!

**Comment by nicolastarzia** [comment]
Thanks @damieng.

**Pull request merged**